### PR TITLE
DTSPO-14838 Setup example flexible server instance

### PR DESCRIPTION
**Pull request by cpareek**
### Change description ###

This is example PR to deploy postgres sql server using ASO.

### Checklist

- [X ] commit messages are meaningful and follow good commit message guidelines

### CODE REVIEW

File: apps/sscs/demo/base/kustomization.yaml [hunk 1]
```diff
@@ -6,8 +6,10 @@ resources:
 - ../../sscs-hearings-api/sscs-hearings-api.yaml
 - ../../sscs-bulk-scan/sscs-bulk-scan.yaml
 - ../../sscs-tribunals-frontend/sscs-tribunals-frontend.yaml
+- ../../../azureserviceoperator-system/resources/flexibleserver-postgres.yaml
```
Comment (timja) [inline apps/sscs/demo/base/kustomization.yaml:9]: missing resource group setup

**Review by timja — CHANGES_REQUESTED** [review]

File: apps/sscs/demo/sops-secrets/demo-postgres.enc.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
```
Comment (timja) [inline apps/sscs/demo/sops-secrets/demo-postgres.enc.yaml:1]: there needs to be instructions somewhere for how to set this up

**Review by timja — CHANGES_REQUESTED** [review]

File: apps/sscs/demo/base/kustomization.yaml (hunk 1 above)
Comment (cpareek) [reply apps/sscs/demo/base/kustomization.yaml:10]: done

File: apps/sscs/demo/sops-secrets/demo-postgres.enc.yaml (hunk 2 above)
Comment (cpareek) [reply apps/sscs/demo/sops-secrets/demo-postgres.enc.yaml:1]: https://github.com/hmcts/cnp-flux-config/blob/master/docs/secrets-sops-encryption.md 
I have initially followed this instructions

Comment (timja) [reply apps/sscs/demo/sops-secrets/demo-postgres.enc.yaml:1]: that should be linked to in the chart documentation, with probably an example command

Comment (cpareek) [reply apps/sscs/demo/sops-secrets/demo-postgres.enc.yaml:1]: https://github.com/hmcts/chart-postgresql/pull/4

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
Closing as it's just an example

**Pull request closed without merging**